packages/investment-pool/contracts/InvestmentPool.sol

```diff
@@ -533,52 +583,46 @@ contract InvestmentPool is
         _afterMilestoneStreamTermination(_milestoneId, streamedAmount, true);
     }
 
-
     // TODO: Decide if needed
-    /** @notice Stops fundraiser campaign
+    /**
+     * @notice Stops fundraiser campaign
      */
-    function cancel()
-        external
-        onlyCreator
-        isNotCanceled
-        fundraiserNotStartedYet
-    {
+    function cancel() external onlyCreator isNotCanceled fundraiserNotEndedYet {
         emergencyTerminationTimestamp = (uint48)(block.timestamp);
         emit Cancel();
     }
 
-
     // TODO: Decide if needed
     /** @notice Stops fundraiser campaign
      */
-    function emergencyCancel()
-        external
-        isNotCanceled
-    {
+    function emergencyCancel() external isNotCanceled {
         emergencyTerminationTimestamp = (uint48)(block.timestamp);
 
-        (uint256 timestamp, int96 flowRate, , ) = cfaV1Lib.cfa.getFlow(acceptedToken, address(this), creator);
+        (uint256 timestamp, int96 flowRate, , ) = cfaV1Lib.cfa.getFlow(
+            acceptedToken,
+            address(this),
+            creator
+        );
 
         // Flow exists, do bookkeeping
         if (timestamp != 0) {
-            uint256 streamedAmount = (_getNow() - timestamp) * (uint256)((int256)(flowRate));
-
-            milestones[_getCurrentMilestoneIndex()].paidAmount += streamedAmount;
+            uint256 streamedAmount = (_getNow() - timestamp) *
+                (uint256)((int256)(flowRate));
 
+            milestones[_getCurrentMilestoneIndex()]
+                .paidAmount += streamedAmount;
 
             cfaV1Lib.cfa.deleteFlow(acceptedToken, address(this), creator, "");
-
         }
 
         emit Cancel();
     }
 
-    function milestoneJump() external isNotCanceled {
-
+    function milestoneJump() external onlyCreator isNotCanceled {
```

Comment:
Perhaps we should add last milestone handling, both here and in Claim function?
There are 2 options for that:

Ether writing some state marker, once we have done a final termination for the las milestone, or perhaps a more preferred way - modifiers for claim/jump methods that can check if an attempt to claim is a valid one.